Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -192,12 +192,13 @@ namespace AzToolsFramework::Prefab
         }
 
         // Close all container entities in the old path.
-        CloseInstanceContainers(m_instanceFocusHierarchy);
+        SetInstanceContainersOpenState(m_instanceFocusHierarchy, false);
 
-        AZ::EntityId previousContainerEntityId = m_focusedInstanceContainerEntityId;
+        RootAliasPath previousContainerRootAliasPath = m_focusedInstanceRootAliasPath;
+        InstanceOptionalReference previousFocusedInstance = GetInstanceReferenceFromRootAliasPath(previousContainerRootAliasPath);
 
         // Do not store the container for the root instance, use an invalid EntityId instead.
-        m_focusedInstanceContainerEntityId = focusedInstance->get().GetParentInstance().has_value() ? focusedInstance->get().GetContainerEntityId() : AZ::EntityId();
+        m_focusedInstanceRootAliasPath = focusedInstance->get().GetAbsoluteInstanceAliasPath();
```

Comment:
As a follow up task, I recommend getting some benchmarks in place for focusing on a prefab. Particularly for a deeply nested prefab, where crawling all the way to the root every time can be expensive. I also recommend consulting with the Technical Artists about what a maximum number could be for a large depth of nesting before writing the benchmarks.